src/coreclr/inc/clrconfigvalues.h

```diff
@@ -735,6 +735,10 @@ RETAIL_CONFIG_STRING_INFO(EXTERNAL_DOTNET_DiagnosticPorts, W("DiagnosticPorts"),
 RETAIL_CONFIG_STRING_INFO(INTERNAL_LTTngConfig, W("LTTngConfig"), "Configuration for LTTng.")
 RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_LTTng, W("LTTng"), 1, "If COMPlus_LTTng is set to 0, this will prevent the LTTng library from being loaded at runtime")
 
+//
+// Executable code
+//
+RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableWXORX, W("EnableWXORX"), 1, "Enable W^X for executable memory.");
```

Comment:
This will be flipped to default to 0 after the CI is green.

Reply:
perhaps a better name for this? `EnableWXORX` is difficult to parse visually.. something like `EnableWriteXorEx`?

Reply:
Makes sense.

Reply:
Renamed to `EnableWriteXorExecute`